src/backend/core/api/viewsets.py

```diff
@@ -363,13 +354,6 @@ def get_queryset(self):
                 queryset.filter(
                     team__accesses__user=self.request.user,
                 )
-                .prefetch_related(
-                    Prefetch(
-                        "user__identities",
-                        queryset=models.Identity.objects.filter(is_main=True),
-                        to_attr="main_identity",
-                    )
-                )
                 # Abilities are computed based on logged-in user's role and
                 # the user role on each team access
                 .annotate(user_role=Subquery(user_role_query))
```

Comment:
Fyi, when rebasing with main, you will pull my recent changes on this precise queryset, where I added the ordering.